### PR TITLE
Ansible tower - ad-hoc command launch playbook fixes

### PR DESCRIPTION
**Pull request by idovandijk**
## Status
Ready

## Related Issues
Fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-4090
Related to: https://jira-hq.paloaltonetworks.local/browse/CIAC-3628

## Description
- Fixed a bug where the playbook input for arguments of the ad-hoc command were not used.
- Fixed a bug where the configuration for the polling playbook was not passed from the playbook inputs.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
5.0.0

## Does it break backward compatibility?
No

### CODE REVIEW

File: Packs/AnsibleTower/Playbooks/Ansible_Tower-_ad_hoc_command-_generic_polling.yml [hunk 1]
```diff
@@ -302,7 +307,7 @@ inputs:
   playbookInputQuery:
 - key: Timeout
   value:
-    simple: "15"
+    simple: "10"
```
Comment (tomer-pan) [inline Packs/AnsibleTower/Playbooks/Ansible_Tower-_ad_hoc_command-_generic_polling.yml:310]: @idovandijk Why did you change the timeout from 15 to 10 as the default value?

Comment (idovandijk) [reply Packs/AnsibleTower/Playbooks/Ansible_Tower-_ad_hoc_command-_generic_polling.yml:310]: The playbook input was set to 15 but the task was hardcoded to use 10, so in order to preserve BC, I changed the input to be the same value that previously was hardcoded (10)

**Comment by idovandijk** [comment]
@tomer-pan need ur approval again

**Review by tomer-pan — APPROVED** [review]
